src/layouts/Titled/components/TOC.tsx

```diff
@@ -121,6 +121,9 @@ export const TOC: React.FC<{ data: IHeading[] }> = ({ data }) => {
         position: sticky;
         top: ${p => p.theme.space.lg};
         margin-left: ${p => p.theme.space.base * 15}px;
+        max-height: calc(100vh - 64px);
```

Comment:
Please use theme values and calculations when they are available. In this case the 64px is 2X the value of `top` coming from `p.theme.space.lg`.

Reply:
Would `p.theme.space.base * 16` be okay? `p.theme.space.lg` is a string value that includes "px".

Reply:
You have the same go-to's we use all over `react-components` such as using polished `math`.